tests/AutoReview/FixerTest.php

```diff
@@ -231,4 +233,13 @@ private static function assertStringIsNotEmpty($actual, $message = '')
         self::assertInternalType('string', $actual, $message);
         self::assertNotEmpty($actual, $message);
     }
+
+    /**
+     * @param mixed  $actual
```

Comment:
I would either write `@param string $actual` or add a type check in the method.

Reply:
Fixed, obviously should be `string` (I've copied it from the method above and missed that).